Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -1281,16 +1257,12 @@ extension ApolloCodegenConfiguration.FileOutput {
     self.schemaTypes = schemaTypes
     self.operations = operations
     self.testMocks = testMocks
-    if let operationIdentifiersPath {
```

Comment:
Need to change the deprecation message on this function, as it doesn't have the `operationManifest` param anymore.

I don't think we need to deprecate that function, as it existed for a short time and was really never used by anyone other than those in the preview (if at all).

Reply:
Shouldn't we still deprecate this because we want users to move from using this `operationIdentifiersPath` to using the new `operationManifestConfiguration`?

Reply:
Yes, but the deprecation message points to an initializer name that has changed. `operationManifest` is no longer a param there.

Reply:
Ah yea good catch